jenkins/jdbc-stage-maven-release.jenkinsfile

```diff
@@ -0,0 +1,80 @@
+lib = library(identifier: 'jenkins@main', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build.git',
+]))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2'
+            args '-e JAVA_HOME=/opt/java/openjdk-11'
+            alwaysPull true
+        }
+    }
+    triggers {
+        GenericTrigger(
+            genericVariables: [
+                [key: 'ref', value: '$.ref'],
+                [key: 'VERSION', value: '$.ref', regexpFilter: 'refs/tags/v' ],
+            ],
+            tokenCredentialId: 'jenkins-sql-generic-webhook-token',
```

Comment:
We have added secret token in jenkins now.
For jdbc use `jenkins-sql-jdbc-generic-webhook-token`.

Reply:
Fixed in e8d9a435, thanks.